v1/client_test.go

```diff
@@ -50,6 +60,386 @@ func TestWriteSchemaCall(t *testing.T) {
 	definition user {}
 	`
 
-	_, err = client.SchemaServiceClient.WriteSchema(context.Background(), &v1.WriteSchemaRequest{Schema: schema})
+	writeResponse, err := client.SchemaServiceClient.WriteSchema(context.Background(), &v1.WriteSchemaRequest{Schema: schema})
+	require.NoError(err)
+	require.NotEmpty(writeResponse.WrittenAt.String())
+
+	readResponse, err := client.SchemaServiceClient.ReadSchema(context.Background(), &v1.ReadSchemaRequest{})
+	require.NoError(err)
+	require.Contains(readResponse.SchemaText, "definition document")
+	require.Contains(readResponse.SchemaText, "definition user")
+}
+
+func TestSchemaWithCaveats(t *testing.T) {
+	t.Parallel()
+	require := require.New(t)
+	client := SutClient(t)
+	defer client.Conn.Close()
+
+	err := WriteTestSchema(client)
+	require.NoError(err)
+}
+
+func TestCheck(t *testing.T) {
```

Comment:
I'm wondering if these should be `Example`s and not `Test`s?

Reply:
`Example`s get run with `go test` they just also show up in the docs as examples